JAVA_CONTRIB_VERSION

```diff
@@ -1 +1 @@
-v1.15.0
```

Comment:
thats a jump, what's this file for?

Reply:
It tracks the version of the JMX jar, used by the JMX receiver; We haven't updated in a while, but we really should be keeping up to date.

Reply:
Also, to clarify, the reason why we moved to update it today is because a vulnerability scan found a few out-of-date dependencies in that jar, and we're looking to rectify that.